content/docs/tasks/security/auth-sds/index.md

```diff
@@ -58,12 +63,9 @@ sleep.bar to httpbin.foo: 200
 sleep.bar to httpbin.bar: 200
 {{< /text >}}
 
-## Verifying no secret-mount file is generated
+## Verifying no secret-volume mounted file is generated
 
-As mentioned, instead of using secret-mount file, key/certificate provisioned by SDS flow are stored in workload sidecar's memory,
-which is more secure and avoids traffic interruption.
-
-Verify no secret-mount file is generated by remotely accessing to workload sidecar container that we just deployed:
+Verify no secret-volume mounted file is generated by remotely accessing to workload sidecar container that we just deployed:
```

Comment:
The text became convoluted and difficult to follow with the added adjective. To simplify the language to make the text more understandable I suggest:
```suggestion
To verify that no secret-volume mounted file is generated, remotely access the deployed workload sidecar container:
```